CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Unreleased
-(None)
+- Remove cluster provider from schema
```

Comment:
Maybe we need a bit more explanation for the result of the change? Something along the lines of:
```suggestion
- Due to https://github.com/pulumi/pulumi/issues/7012 including the provider in the generated SDK never really worked. This is removed now. Note - existing uses in Nodejs should not be affected.
```